app/assets/stylesheets/dashboard.css.scss

```diff
@@ -1,10 +1,78 @@
+$circle-color: rgba($primary-color,0.5);
+
+.dash-header {
+  background-color: $topbar-bg-color;
+}
+
+.avatar-preview img {
+  padding: 1em;
+}
+
 .circle-thumb {
-  background-color: rgba(125,0,255,0.5);
-  border-radius: 10px;
+  background-color: $circle-color;
+  border-radius: 7px;
   box-sizing: border-box;
-  padding: 1em;
+  height: 200px;
+  overflow: hidden;
+  position: relative;
+
+  .circle-name {
+    width: 100%;
+    position: absolute;
+    bottom: 0px;
+  }
+
+  .circle-pic img {
+    width: 100%;
+    @extend .hvr-grow;
+  }
+
+  div.circle-pic {
+    position: absolute;
+  }
+
+  div.circle-texts {
+    width: 100%;
```

Comment:
Properties should be ordered background-color, border-radius, color, height, opacity, padding, position, -moz-transition, -webkit-transition, width